content/en/blog/sysflow-falco-sidekick.md

```diff
@@ -0,0 +1,146 @@
+
+<br/>
+
+![image description](/img/sysflow-falco-sidekick/01-diagram-outputs.png)
+
+<br/>
+
+### What challenges does it solve?
+
+These are the keys that sets SysFlow apart:
+
+* It drastically **reduces data footprints** when compared to raw system call collection.
+* **Reduces event fatigue** (a.k.a. "too many alerts") thanks to its underlying entity-relational model and flow abstractions.
+* At the same time, it **minimizes the requirements for hard filters**.
+* It **provides useful context** by linking together system event data at the data format level. This speeds up security analysis, enabling analytics beyond simple manual policies, including machine learning, stateful analytics, and automated policy generation.
+* It treats system security monitoring as a data science problem through **rapid edge analytics building**, easy-to-use APIs, and support for multiple data serialization and export formats.
+
+
+### Built on Falco libs
+
+The SysFlow format is supported by an [edge processing pipeline](https://github.com/sysflow-telemetry/sf-processor) that aggregates event provenance information and supports real-time enrichment of the telemetry stream with attack TTP labels, environment metadata, and Kubernetes log data.
+
+As a benefit of using the Sysflow format, users go from managing individual events generated from different sources to obtaining an enhanced dataset to work with, enabling them to focus on writing and sharing analytics on a scalable, common open-source platform.
+
+The framework builds on [Falco libs](https://github.com/falcosecurity/libs) and the [Falco rules language](https://falco.org/docs/rules/) to create the plumbing required for system telemetry as shown in the diagram below.
+
+Falco libs facilitate the collection of system events generated from the host and containers through the eBPF probe in the kernel space.
+
+At user level, The SysFlow Collector, also leveraging Falco libs, collects the previous events and exports the captured information using Apache Avro object serialization
+
+The SysFlow Processor processes and enriches the data. The policy engine applies logical rules to filter, alert or semantically label sysflow records using a declarative language based on Falco rules.
+
+<br/>
+
+![image description](/img/sysflow-falco-sidekick/02-sysflow-falco-libs.png)
+
+<br/>
+
+The SysFlow framework is designed as a pluggable edge processing architecture, which includes a policy engine, and what's more noteworthy, an experimental graph engine.
+
+The policy engine accepts declarative [policies](https://sysflow.readthedocs.io/en/latest/processor.html#writing-runtime-policies) (written in Falco rules syntax) and supports edge filtering, tagging, and alerting on SysFlow streams.
+
+And a really flashy addition is the experimental graph engine, that clusters and exports concise behavioral profiles of monitored applications. We'll see an example in a moment.
```

Comment:
And a visual addition is the experimental graph engine, which clusters and exports concise behavioral profiles of monitored applications. We'll see an example in a moment.